plot/plot.app.js

```diff
@@ -628,7 +632,14 @@
                             } else {
                                 plot.traces.forEach(function (trace) {
                                     var uri = trace.uri;
-                                    server.http.get(uri).then(function(response) {
+                                    var headers = {};
+                                    headers[ERMrest.contextHeaderName]=ConfigUtils.getContextHeaderParams();
+                                    if(UriUtils.getQueryParams($window.location.href).pcid)
+                                        headers[ERMrest.contextHeaderName].pcid=UriUtils.getQueryParams($window.location.href).pcid;
+                                    if(UriUtils.getQueryParams($window.location.href).ppid)
+                                        headers[ERMrest.contextHeaderName].ppid=UriUtils.getQueryParams($window.location.href).ppid;
+
+                                    server.http.get(uri,{ headers: headers }).then(function(response) {
```

Comment:
Same comment as above, we want to call `ERMrest._certifyContextHeader()` before this get request.

Reply:
made required changes